decidim-proposals/app/forms/decidim/proposals/proposal_form.rb

```diff
@@ -16,6 +16,10 @@ class ProposalForm < Decidim::Proposals::ProposalWizardCreateStepForm
       attribute :has_address, Boolean
       attribute :attachment, AttachmentForm
```

Comment:
Should we remove this attribute now?

Reply:
yes I think we can remove it. I'll do that. Thanks

Reply:
@tramuntanal after trying to remove the attribute I realize that it's used by the collaborative drafts.

Reply:
Ah, ok then